app/lib/git_repository.rb

```diff
@@ -62,7 +62,7 @@ def self.do_commit(repo, author, message)
     index = repo.index
     commit_tree = index.write_tree(repo)
     index.write
-    commit_author = {email: 'markus@markus.com', name: author, time: Time.now}
+    commit_author = { email: 'markus@markus.com', name: author, time: Time.current }
```

Comment:
Looks like this change might be causing an error in the git tests (see Travis report). Please change back for now, we can look into this later.

Reply:
Oh, yes! Sorry about that, I thought this change would be fine... Let me change back now.

Reply:
Oh, no the issue isn't the spaces, it's the `Time.now` vs. `Time.current` (I think). You should change this back to `Time.now`.

Reply:
I've just changed this back to `Time.now`, hopefully this time it will work.

Reply:
All checks have passed according to Travis, yeah. I'm a little confused why`Time.now` is the only correct version in this case.